moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_states.cpp

```diff
@@ -245,8 +245,21 @@ void MotionPlanningFrame::removeStateButtonClicked()
 
 void MotionPlanningFrame::clearStatesButtonClicked()
 {
-  robot_states_.clear();
-  populateRobotStatesList();
+  QMessageBox msg_box;
```

Comment:
Why introducing a security message box, if the data is only removed from memory?

Reply:
Confirmation dialogues in front of destructive operations are just prudent, in my opinion. You never know how many states the user has stored, or if they misunderstood the button. Consider non-native speakers, too.

Reply:
For me, confirmation dialogues are just annoying :wink: